disparse-core/src/main/java/disparse/parser/dispatch/CommandRegistrar.java

```diff
@@ -303,12 +303,10 @@ private boolean isOnCooldown(Command command, AbstractDispatcher<E, T> helper, E
       String cooldownMessage = scope.getCooldownMessage();
       Duration left = cooldownManager.timeLeft(cooldownCompositeKey, cooldownDuration);
 
-      if (!left.isZero()) {
-        if (command.isSendCooldownMessage()) {
-          helper.sendMessage(event, cooldownMessage);
-        }
-        return true;
+      if (!left.isZero() && command.getCooldownMessage().equals(CooldownMessage.ENABLED)) {
```

Comment:
This seems to mimic the current behavior, just hidden behind an Enum now which is pretty close.  But where do we actually handle the case where it's CooldownMessage.REACTION?  I might be missing it.

Reply:
Well, I think more importantly I'll need to understand Toby's point in the first comment made to #26 raised issue. Can you help to tell from the business point of view what is the usage of "reaction - invocation command receives a reaction with "🕒" indicating the command is on cooldown." ?

Reply:
Sure, so if it's enabled we want to send the message it currently sends.  Such as "X command is on cooldown for Y seconds!" or something.

If it's disabled, we want to not send anything.

The enabled case can be quite noisy.  Imagine someone spamming a command on cooldown forcing the bot to continuously say it's on cooldown.  So this third option `REACTION` will instead of sending a message the bot would react on the message with the Clock Emoji.  This would be less noisy, but still give some indication that the command was on cooldown to the user.

Also, by using this enum it will let us do even more strategies in the future.  So changing this out is good for future changes we might make too.  

So you would probably have to make some method 'reactToMessage' which takes the event, and then implement it for D4J, JDA, and small-d.  If you want to discuss this more you could add me on Discord Bosco#8564 and we could hash it out.